projects/core/src/cms/page/routing/page-links-meta.resolver.ts

```diff
@@ -0,0 +1,81 @@
+import { Injectable } from '@angular/core';
+import { NavigationStart, Router } from '@angular/router';
+import {
+  CanonicalUrlOptions,
+  PageMetaConfig,
+  WindowRef,
+} from '@spartacus/core';
```

Comment:
[medium] circullar dep on spartacus/core
```suggestion
import { WindowRef } from '../../../window/window-ref';
import {
  CanonicalUrlOptions,
  PageMetaConfig,
} from '../config/page-meta.config';
```